sepal_ui/mapping/mapping.py

```diff
@@ -782,3 +784,22 @@ def get_viz_params(image):
                     props = {}
 
         return props
+
+    def add_layer(self, layer, default_hover=False):
```

Comment:
what about changing `default_hover` to a simpler `hover` parameter ?

Reply:
I've actually changed from `hover` to `default_hover` in my last commit... I agree that it can be useless.